From 0e6659ec960734b0b01aad196d4bdb4a3800b493 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Thu, 16 Oct 2014 19:10:59 +0200
Subject: [PATCH] Revert "Missing initialization for the catalog module"

It's not correct to always load the default catalog.
https://bugzilla.redhat.com/show_bug.cgi?id=1153753

This reverts commit 054c716ea1bf001544127a4ab4f4346d1b9947e7.

---
 parser.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/parser.c b/parser.c
index 1d93967..67c9dfd 100644
--- a/parser.c
+++ b/parser.c
@@ -14830,9 +14830,6 @@ xmlInitParser(void) {
 #ifdef LIBXML_XPATH_ENABLED
 	xmlXPathInit();
 #endif
-#ifdef LIBXML_CATALOG_ENABLED
-        xmlInitializeCatalog();
-#endif
 	xmlParserInitialized = 1;
 #ifdef LIBXML_THREAD_ENABLED
     }
-- 
1.9.3
